Sources/BranchSDK/BranchLogger.m

```diff
@@ -77,8 +93,10 @@ - (void)logMessage:(NSString *)message withLevel:(BranchLogLevel)level error:(NS
     if (self.includeCallerDetails) {
         formattedMessage = [NSString stringWithFormat:@"%@ %@", [self callingClass], message];
     }
-
-    if (self.logCallback) {
+    
+    if  (self.advancedLogCallback) {
```

Comment:
@nsingh-branch  Since `advancedLogCallback` is always non-null, its initialized in `init`, `else if ` condition will never be executed.

Reply:
I removed the default init since it shouldn't be necessary to have both. All of the tests appear to be passing now.

Reply:
Did the new automation succeed as well?